src/sctools/test/test_count.py

```diff
@@ -100,6 +100,34 @@ def __repr__(self):
                f'{consts.GENE_NAME_TAG_KEY}: {self.gene_name}'
 
 
+class CellMoleculeGeneQueryNameSortOrder(bam.AlignmentSortOrder):
```

Comment:
I think this class should be in the same file as `AlignmentSortOrder` and `QueryNameSortOrder()`, whether that's here or in `bam.py`